### ISSUE TITLE
orders totally out of range of balance.

### ISSUE THREAD

**Issue by Tarexx** [jufkes/hitbtcBot#1]
I was trying to use your bot to trade BTC/ETH but somehow it always only places orders of 1000 ETH no matter what balance i have and also no matter what round values i use in the script.

**Comment by jufkes** [comment]
Yeah i had to do some hacky shiz to get this to work with TRX (what i initially built it for). It would only accept orders in rounded 1k increments. It was really disappointing.

Due to that and some other issues with hitBTC i've pulled all of my tokens from it and will likely not return. Thus, I cannot test.

I'll be turning this repo to private and removing this bot from the list. You're welcome to fork the repo and try to work through their ordering bs. 

I'll leave it up for 24 hours.

**Comment by Tarexx** [comment]
I just have like 50 DENT on there so i don't really wanna bother spending days of trial and error :D but i really like your bots, they got quite some clean profit % when set up properly :)

**Comment by Tarexx** [comment]
I sure as hell will help improving the other ones, especially the kucoin bot since kucoin is just fairly new and there's alot of potential in this exchange

**Comment by jufkes** [comment]
I'm glad to hear it! I'm happy i can help us all make a little more coin. It's a simple strategy that takes advantage of the volatility without having to spend hours staring at charts (which i do anyway :-D)

I'm botting UTK on kucoin and it's been absolutely magical. If you have any features or changes i can do there i'm all ears.

Starting one for binance as well.

**Comment by jufkes** [comment]
Oh, UTK, my settings are:

buyPercent 6
sellPercent 6
volume 6
buyDifference 2

**Comment by Tarexx** [comment]
i'm atm trading on BTC: BCD,DENT,ETH,HSR, KNC and ETH: BCD,DENT, HR,KNC because i just started with like 10 bucks but my phone is going ham with sell/buy messages and my balance is just going up atleast 15% a day :D i got pretty much the same settings for all these pairs and havent noticed any high buy / sell low or any other issue in that manner. as for features i feel like some regular balance/profit announcements per session seem reasonable, for when you are not at your computer. Going a bit more advanced maybe an algorithm that looks for the best pair to trade at the moment and then automatically buys in to that and trades on that till a certain profit is reached / the price crashes / whatever else reason there is to stop trading.

**Comment by jufkes** [comment]
For profit tracking, I work with the guys who built cryptonotify (and contribute to). They should be adding support for all these additional exchanges in the new version coming soon. That should give you the ability to regularly track how your investments are doing. 
If you're on bittrex or polo, absolutely give it a whirl! 

I've been kicking around the possibility of doing a bot that will skim an exchange and estimate on mac'd on the 1hr. It's a mathematical and architectural nightmare though. Not to mention risky as hell. 

If you havent looked at atomsignal.com, that's one that I just recently started using (4 days ago lol) since reading the charts for all exchanges is too time consuming. It sends an "open" (buy) and "close" (sell) recommendation with it's profit tracking. I believe it's $60/mo though. When something signals, I check the MACd trend on tradingview...if it's trending up I buy in. It's a little early to tell on it's overall profitability...especially with the BTC drop
https://www.atomsignal.com?ref=290286

**Comment by Tarexx** [comment]
I've been looking also into some new crypto-pump bot, it basically works on messages on Telegram to determine when to buy in and when to drop out. There seem to be quite the community around pumping so maybe looking into that might be a great idea. As for the idea i was having, surely it would also require quite the computing power to keep watching like 200+ different charts per exchange. I'm definitely gonna check out atomsignal, seems to be what i was looking for, thx for the recommendation. Does cryptonotifiy have an API ? might look into fiddling with that then.
